composer.json

```diff
@@ -11,10 +11,10 @@
     ],
     "require": {
         "php": ">=5.4",
-        "ext-mongodb": "^1.2.0"
+        "ext-mongodb": ">=1.2.0"
```

Comment:
This use of the [caret operator](https://getcomposer.org/doc/articles/versions.md#caret) is intentional. The library supports extension versions >=1.2.0 and <2.0.0, as 2.0.0 could include BC breaks. It's also not clear to me how this change addresses the error you shared, as the most recent extension release is currently 1.2.9 (see [PECL](https://pecl.php.net/package/mongodb)).